doc/release-notes/release-notes-0.3.1.md

```diff
@@ -0,0 +1,22 @@
+Tapyrus Signer version 0.3.1 is now available for download at: 
+  https://github.com/chaintope/tapyrus-signer/release/tag/v0.3.1/
+    
+Please report bugs using the issue tracker at github:
+  https://github.com/chaintope/tapyrus-signer/issues
+  
+Project source code is hosted at github; you can get
+source-only tarballs/zipballs directly from there:
+  https://github.com/chaintope/tapyrus-signer/tarball/v0.3.1  # .tar.gz
```

Comment:
I could not download from this url.

Reply:
Yes, i've not created yet. I'm going to create binary archive and tag after RP #52, #53 will be merged.